examples/demoslave.py

```diff
@@ -0,0 +1,37 @@
+from pythonfmu import Fmi2Causality, Fmi2Variability, Fmi2Slave, Boolean, Integer, Real, String
+
+slave_class = "Resistor"  # REQUIRED - Name of the class extending Fmi2Slave
+
+
+class Resistor(Fmi2Slave):
+
+    author = "John Doe"
+    description = "A simple description"
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+        self.positive_pin_v = 20.
+        self.positive_pin_i = 0.001
+        self.negative_pin_v = 10.
+        self.negative_pin_i = 0.001
+        self.delta_v = 10.
+        self.i = 0.001
+        self.R = 10000.
+        
+        self.register_variable(Real("R", causality=Fmi2Causality.parameter))
```

Comment:
This parameter on the other hand should be marked `tunable` as you state users may change it.